go

```diff
@@ -44,63 +44,78 @@ IMAGES = %w(
   team-api
 )
 
-def message_marker(message)
-  marker = '-' * message.size
-  "#{marker}\n#{message}\n#{marker}"
-end
+DATA_CONTAINERS = {
+  'pages-data' => 'pages'
+}
 
-def build_image(image)
-  puts message_marker("Building #{image}")
-  exec_cmd "docker build -t #{image} -f ./#{image}/Dockerfile ./#{image}"
-end
+DAEMON_TO_DATA_CONTAINERS = {
+  'nginx-18f' => ['pages-data:ro'],
+  'pages' => ['pages-data:rw'],
+}
 
 def_command :build_images, 'Build Docker images' do |args|
-  (args.empty? ? IMAGES : args).each { |image| build_image(image) }
+  (args.empty? ? IMAGES : args).each do |image|
+    message = "Building #{image}"
+    marker = '-' * message.size
+    puts "#{marker}\n#{message}\n#{marker}"
+    exec_cmd "docker build -t #{image} -f ./#{image}/Dockerfile ./#{image}"
+  end
 end
 
-def _remove_container(image_name)
-  exec_cmd "if $(docker ps -a | grep -q ' #{image_name}$'); then " \
-    "docker rm #{image_name}; fi"
+def_command :create_data_containers, 'Create Docker data containers' do |args|
+  (args.empty? ? DATA_CONTAINERS.keys : args).each do |container_name|
```

Comment:
if someone specifies `args` and it's not the ones specified in `DATA_CONTAINERS` we might get some weird  stuff such as an empty base_image name